src/components/AgreementLineSections/EresourceSelector/EresourceSelector.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import '@folio/stripes-erm-components/test/jest/__mock__';
+import { renderWithIntl } from '@folio/stripes-erm-components/test/jest/helpers';
+import { FormattedMessage } from 'react-intl';
+import translationsProperties from '../../../../test/helpers';
+import EresourceSelector from './EresourceSelector';
+
+const line = {
+  'component': 'ƒ EresourceSelector() {}',
+  'error': '<Memo />',
+  'name': 'linkedResource',
+  'onAdd': () => {},
+  'value': ''
+};
+
+describe('EresourceSelector', () => {
+  let renderComponent;
+  beforeEach(() => {
+    renderComponent = renderWithIntl(
+      <EresourceSelector error={<FormattedMessage id="ui-agreements.agreementLine.provideEresource" />} line={line} />,
+      translationsProperties
+    );
+  });
+
+  test('renders eholdings card component', () => {
+    const { getByText } = renderComponent;
+    expect(getByText('eHoldings')).toBeInTheDocument();
+   });
+
+   test('renders linkEresourceToStart message', () => {
+    const { getByText } = renderComponent;
+    expect(getByText('Link an e-resource to get started')).toBeInTheDocument();
+   });
+
+   test('renders error message', () => {
```

Comment:
This test needs to be broken into a different describe block. This block should deal with rendering an EresourceSelector component with a line. The second describe block should also pass in an error. Lets not mix up both the tests into one block